firebase-messaging/src/main/java/com/google/firebase/messaging/RemoteMessage.java

```diff
@@ -373,13 +368,13 @@ public String getMessageType() {
     /** @hide */
     @Nullable
     public String getCollapseKey() {
-      return bundle.getString(MessagePayloadKeys.MESSAGE_TYPE);
+      return bundle.getString(MessagePayloadKeys.COLLAPSE_KEY);
     }
 
     /** @hide */
     @IntRange(from = 0, to = 86400)
     public int getTtl() {
-      return Integer.parseInt(bundle.getString(MessagePayloadKeys.MESSAGE_TYPE, "0"));
```

Comment:
Is this a bugfix?

Reply:
Yes, sort of. They're hidden and not actually used anywhere (which is why it wasn't broken and why the change doesn't require any test changes).